docs/components/utilities.md

```diff
@@ -187,10 +187,16 @@ Float an element to the left or right with a class. `!important` is included to
 
 ## Center content
 
-Set an element to `display: block;` and center via `margin`. Available as a mixin and class.
+Center an element via `margin`, center-block for most use cases, center-tables for tables. Available as mixins and classes.
```

Comment:
References to classes (in this case, `.center-block` and `.center-table`) should be formatted as
```
`.someclass`
```